Sphinxsetup.sh

```diff
@@ -7,27 +7,41 @@ if [ "$UID" -gt 0 ]; then
      exit 1
 fi
 
-DISTRIBUTION_CODENAME=$(lsb_release -i -s)
-if [ ${DISTRIBUTION_CODENAME} == 'Ubuntu' ]; then
-  add-apt-repository universe
+DISTRIBUTION_ID=$(lsb_release -i -s)
+if [ ${DISTRIBUTION_ID} == 'Ubuntu' ]; then
```

Comment:
This isn't good as it is working perfectly on debian

Reply:
Fixed.